src/App.css

```diff
@@ -36,3 +36,26 @@
     transform: rotate(360deg);
   }
 }
+
+.station-wrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+
+.dropdown-selections {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 10px;
+}
+
+.dropdown-selections > div {
+  margin: 10px;
+}
+
+.button {
+  height: auto;
+}
```

Comment:
we want to be adding a newline at the end of all files, as it is a styling convention, github gives this angry red circle when you forget so it's quick to spot when you're looking at your changes